index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="chrome=1">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://unpkg.com/leaflet@1.7.1/dist/leaflet.css"
+    integrity="sha512-xodZBNTC5n17Xt2atTPuE1HxjVMSvLVW9ocqUKLsCC5CXdbqCmblAshOMAS6/keqq/sMZMZ19scR4PsZChSR7A=="
+    crossorigin=""/>
+    <script src="main.js"></script>
+    <script src="https://unpkg.com/leaflet@1.7.1/dist/leaflet.js"
+    integrity="sha512-XQoYMqMTK8LvdxXYG3nZ448hOEQiglfqkJs1NOQV44cWnUrBc8PkAOcXy20w0vlaXaVUearIOBhiXZ5V3ynxwA=="
+    crossorigin=""></script>
+    <title>Leaflet</title>
+    <style>
+    #map { 
+    height: 580px;
+    width: 580px;
+    background-color: #ccc;
+    }
+    
+    #map::before{
+        content: 'LOADING';
+        font-size: 3em;
+        text-align: center;
+        width: 100%;
+        position: absolute;
+        margin-top: 3em;
+}
+    </style>
+</head>
+<body>
+    <body>
+        <form>
+            <select id="business">
+                <option value="business">Businesses</option>
+                <!-- <option value="coffee">Coffee Shops</option>
+                <option value="hotel">Hotels</option>
+                <option value="grocery">Grocery Stores</option> -->
+            </select>
+            <button id="submit">Submit</button>
```

Comment:
The issue with the page reloading is because the button is trying to submit :( If you add `type="button"` in your first button tag it won't do that anymore -- then you can see that clicking the submit button successfully adds markers! :D (So you'd just need to remove the ones that load on page load and you'd be all set)